ecl/hthor/hthor.cpp

```diff
@@ -6283,6 +6283,7 @@ const void * CHThorAllJoinActivity::nextRow()
                     return ret;
                 }
             }
+            break;
```

Comment:
Interesting! Looks like this missing break actually (by luck) has no ill effect. But worth fixing!